src/include/network/lan/address.rb

```diff
@@ -1621,5 +1602,19 @@ def initial_hostname(ipaddr)
 
       String.FirstChunk(Ops.get(host_list, 0, ""), " \t")
     end
+
+    def firewall_zones
+      @firewalld ||= Y2Firewall::Firewalld.instance
```

Comment:
NP: Just a suggestion, why not creating a method like this?
```ruby
def firewalld
  @firewalld ||= Y2Firewall::Firewalld.instance
end
```

And use it forgetting about initialization. Just thinking ahead...